controllers/operandconfig.go

```diff
@@ -241,29 +250,30 @@ func (r *CommonServiceReconciler) updateOperandConfig(newConfigs []interface{})
 			}
 			if rules != nil && rules.(map[string]interface{})["spec"] != nil && rules.(map[string]interface{})["spec"].(map[string]interface{})[cr] != nil {
 				ruleForCR := rules.(map[string]interface{})["spec"].(map[string]interface{})[cr].(map[string]interface{})
-				opService.(map[string]interface{})["spec"].(map[string]interface{})[cr] = mergeCRsIntoOperandConfig(spec.(map[string]interface{}), newConfigForCR, ruleForCR, overwrite)
+				opService.(map[string]interface{})["spec"].(map[string]interface{})[cr], isChangedWithRules = mergeCRsIntoOperandConfig(spec.(map[string]interface{}), newConfigForCR, ruleForCR, overwrite)
 			} else {
 				if overwrite {
```

Comment:
does this overwrite variable represent whether OperandConfig values should be overwritten? If so, can we not just return this variable to indicate whether there have been updates?